code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+
+import "../index.css"
+
+import BackButton from "../components/BackButton"
+
+export const MovieDetails = () => {
+  const { movieId } = useParams()
+  const [movieDetails, setMovieDetails] = useState([])
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${movieId}?api_key=50eb4adf4920d3714372a2ce134f64ac&language=en-US`
+    )
+      .then((res) => res.json())
+      .then((json) => 
+        setMovieDetails([json])
```

Comment:
We're unsure why you have **[ ]** around json here?  We had it it like **then((json) => setDetails(json));** and it worked fine. Just curious.